### ISSUE TITLE
ctrl+alt keybindings should not be used by default under Windows

### ISSUE THREAD

**Issue by Tyriar** [felipecaputo/git-project-manager#57]
When you start up on Windows you see this in the console:

![image](https://user-images.githubusercontent.com/2193314/29474453-a4cf61ca-8410-11e7-83ed-21aa844aaf04.png)

I'm on Insiders, not sure if that matters.

**Comment by felipecaputo** [comment]
@Tyriar, do you know if I change the default key binding, people that is already using the default bindings will keep then when the extension updates, or they will need too go through the `Ctrl + Shift + P` and search for the command to see the new keybind?

**Comment by Tyriar** [comment]
If you change the keybinding it will everyone who use the extension will have their keybindings change unless they explicitly added their own keybindings. Not sure what the right move is here, but the warning is super annoying personally 😛

**Comment by felipecaputo** [comment]
@Tyriar I was thinking here and implemented a `legacyOpenProject` command, that keeps the old `Ctrl+Alt+P` keybind, but just Do you thing this is a good approach?

**Comment by Tyriar** [comment]
@felipecaputo I think that will still have the same problem of default extension keybindings using alt, no?

**Comment by felipecaputo** [comment]
Yes @Tyriar, but that *legacy* option would be kept for only one version, to avoid changing abruptly.

I was thinking yesterday, maybe this legacy option should open keybings file, to allow user to configure if he wants to keep the old keybing ou use the new one

**Comment by Tyriar** [comment]
@felipecaputo ah, I missed the message. Sounds good :smiley: 

That's how we typically deprecate in VS Code.
